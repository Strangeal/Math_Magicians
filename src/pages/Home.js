import '../App.css';

const Home = () => (
  <section className="home">
    <h1>Welcome to our Page!</h1>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi
      accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi
      aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis
      distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora
      voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque
      architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet
      consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et
      labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi
      accusamus, itaque architecto nemo omnis nobis distinctio?
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi
      accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi
      aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis
      distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora
      voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque
      architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet
      consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et
      labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi
      accusamus, itaque architecto nemo omnis nobis distinctio?
    </p>

  </section>
);

export default Home;
