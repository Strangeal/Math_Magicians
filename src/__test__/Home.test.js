import React from "react";
import { render } from "@testing-library/react";
import Home from "../pages/Home";

it("renders to match the snapshot", () => {
  const tree = render(<Home />);
  expect(tree).toMatchSnapshot();
});
