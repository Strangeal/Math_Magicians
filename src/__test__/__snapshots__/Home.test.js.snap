// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to match the snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="home"
      >
        <h1>
          Welcome to our Page!
        </h1>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio?
        </p>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio?
        </p>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="home"
    >
      <h1>
        Welcome to our Page!
      </h1>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio?
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio?
      </p>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`renders when there is item 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="home"
      >
        <h1>
          Welcome to our Page!
        </h1>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio?
        </p>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio?
        </p>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="home"
    >
      <h1>
        Welcome to our Page!
      </h1>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio?
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio? Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, modi accusamus. Nostrum autem et labore eveniet tempora voluptatem, eligendi aliquam, fugit ipsum nisi accusamus, itaque architecto nemo omnis nobis distinctio?
      </p>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
